package_manifest.yaml

```diff
@@ -414,6 +414,7 @@ foreman_nonscl_packages:
     rubygem-kafo: {}
     rubygem-kafo_parsers: {}
     rubygem-kafo_wizards: {}
+    rubygem-smart_proxy_dhcp_bluecat: {}
```

Comment:
I think we need to update our add script to the updated structure. Could you move it to the `foreman_proxy_plugin_packages` group?

Reply:
Maybe the scripts are ok. I think if you only specify the gem name and the template, they end up in the right category. But The usage message does not suggest that.